src/sweet_bison.h

```diff
@@ -0,0 +1,93 @@
+/* A Bison parser, made by GNU Bison 3.8.2.  */
```

Comment:
We don't want the file generated by lex/bison on the repo. Please add the to the .gitignore